test/README.md

```diff
@@ -102,26 +102,22 @@ go test -v -tags=e2e -count=1 ./test/e2e --resolvabledomain
 The [`upload-test-images.sh`](./upload-test-images.sh) script can be used to build and push the
 test images used by the conformance and e2e tests. It requires:
 
-* [`DOCKER_REPO_OVERRIDE`](/DEVELOPMENT.md#environment-setup) to be set
+* [`KO_DOCKER_REPO`](/DEVELOPMENT.md#environment-setup) to be set
 * You to be [authenticated with your
   `DOCKER_REPO_OVERRIDE`](/docs/setting-up-a-docker-registry.md)
```

Comment:
change this too

Reply:
I changed it back. In `upload-test-images.sh`, it also changed it back to require `DOCKER_REPO_OVERRIDE` so that the doc here can still make sense. But I added `export KO_DOCKER_REPO=${DOCKER_REPO_OVERRIDE}` just like it in `e2e-test.sh`. Let me know if that makes sense.